boltdb/adaptor_test.go

```diff
@@ -0,0 +1,221 @@
+package boltdb_test
+
+import (
+	"context"
+	"errors"
+
+	"github.com/dogmatiq/dogma"
+	"github.com/dogmatiq/enginekit/fixtures"
+	. "github.com/dogmatiq/projectionkit/boltdb"
+	"github.com/dogmatiq/projectionkit/boltdb/internal"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"go.etcd.io/bbolt"
+)
+
+var _ = Describe("MessageHandler Adaptor", func() {
```

Comment:
```suggestion
var _ = Describe("type adaptor", func() {
```

I've taken to `Describe`ing using strings with "go-like" syntax where appropriate.  This is done universally throughout all but the oldest of Dogmatiq tests.